docs/proposals/rename-group.md

```diff
@@ -0,0 +1,156 @@
+- Change the `topolvm.cybozu.com/resize-requested-at` annotation to` topolvm.io/resize-requested-at`
+- Change the `topolvm.cybozu.com/logicalvolume` finalizer to` topolvm.io/logicalvolume`
+
+**Since the CRD itself changes for the Logical Volume resource, it is necessary to delete the old CRD resource and create a new CRD resource when migrating the group name. Also, in order to prevent the deletion of the actual LVM volume due to the logical volume migration, it is recommended to migrate with the TopoLVM components stopped.**
+
+##### Node
+
+Make the following changes for each Node resources
+
+- Change the `topolvm.cybozu.com/node` finalizer to` topolvm.io/node`
+
+##### PersistentVolumeClaim
+
+Modify each PersistentVolumeClaim resources in TopoLVM as follows:
+
+- Change the value of the `volume.beta.kubernetes.io/storage-provisioner` annotation to` topolvm.io`
+- Change the value of the `volume.kubernetes.io/storage-provisioner` annotation to` topolvm.io`
+- Change the `topolvm.cybozu.com/pvc` finalizer to` topolvm.io/pvc`
+
+##### PersistentVolume
+
+Modify each PersistentVolume resource in TopoLVM as follows:
+
+- Change the value of the `pv.kubernetes.io/provisioned-by` annotation to` topolvm.io`
+- Change the value of the `.spec.csi.driver` field to` topolvm.io`
+- Change the string `topolvm.cybozu.com` in the value of the `.spec.csi.volumeAttributes ["storage.kubernetes.io/csiProvisionerIdentity "]` field to` topolvm.io`
+
+**Since it is not possible to change the above fields of the PersistentVolume resource, it is necessary to recreate the resource when migrating the group name. Also, if `.spec.persistentVolumeReclaimPolicy = Delete` is set, there is a risk of deleting the actual LVM volume, so it is recommended to temporarily change it to` Retain` and then delete it.**
+
+### Enable the use of topolvm.cybozu.com
+
+We will make the following changes to allow users to continue to use `topolvm.cybozu.com`.
+
+#### Addition of group name setting function
+
+- If you set the `USE_LEGACY_PLUGIN_NAME` environment variable, make sure that the group name uses` topolvm.cybozu.com`
+- If the `USE_LEGACY_PLUGIN_NAME` environment variable is not set, the group name` topolvm.io` will be used
+
+#### Group name setting in helm chart
+
+Add the `.Values.useLegacyName` variable to the helm chart and set this variable to` true` to set the installation manifest to use `topolvm.cybozu.com`.
+If the `.Values.useLegacyName` variable is not set to` true`, the group name will be `topolvm.io`.
+
+If you have set the following variables, you may need to set them appropriately to use `topolvm.io` or` topolvm.cybozu.com` in the variables.
+
+- `.Values.storageClasses`
+- `.Values.node.volumes`
+- `.Values.volumeMounts.topolvmNode`
+
+#### Automatic generation for API and CRD
+
+Automatically generating the API and CRD of `topolvm.cybozu.com` based on the API and CRD of` topolvm.io`.
```

Comment:
```suggestion
Automatically generating the API and CRD of `topolvm.cybozu.com` based on the API and CRD of `topolvm.io`.
```

Reply:
Also, what did you mean by saying "based on the API and CRD of` topolvm.io`"? I could not imagine how the API and CRD of `topolvm.cybozu.com` are generated automatically.